src/GUI/ExchangeStatus.java

```diff
@@ -150,7 +151,14 @@ public String toString() {
                 bestOrders.put(entry.getKey(), new EnumMap<OrderType, Double>(OrderType.class));
                 try {
                     for (Map.Entry<OrderType, SortedSet<Order>> orderEntry : exchangeServer.getExchange().getOrderStatus(security.getTicker()).entrySet()) {
-                        bestOrders.get(security).put(orderEntry.getKey(), orderEntry.getValue().first().getPrice());
+                        if(orderEntry.getValue().size()>0)
+                        {
```

Comment:
Yo, can you please not change how the brackets are?
Alt+shift+F in Netbeans before you commit please.